data/localizations/categories/en.yml

```diff
@@ -31082,6 +31082,39 @@ en:
     vp-2-2-6:
       name: Snowmobiles
       context: Vehicles & Parts > Vehicles > Motor Vehicles > Snowmobiles
+    vp-2-2-7:
+      name: Motor Sports Vehicles
+      context: Vehicles & Parts > Vehicles > Motor Vehicles > Motor Sports Vehicles
+    vp-2-2-7-1:
+      name: Race Cars
+      context: Vehicles & Parts > Vehicles > Motor Vehicles > Motor Sports Vehicles > Race Cars
```

Comment:
Categories make sense to me. I suggest we add a few additional leaves like GT Cars, Stock Cars or Racing Trucks.